README.md

```diff
@@ -172,11 +172,18 @@ There are 5 different environment tiles, of which some have different styles of
 #### Locations
 As of January 2022, there are a total of 29 different locations in game.
 
-A lot of the earlier details such as what order which locations were unlocked or released in are no longer easily found without restarting the game, however looking at the game's [version history][history], the rough suggested order is:
+A lot of the earlier details such as what order which locations were unlocked or released in are no longer easily found without restarting the game, however looking at the game's [version history][history], the suggested order is:
+
+1. Prague & Warsaw ([1.3](https://www.ipa4fun.com/ipa/64334/))
+2. UK & Scandinavia ([1.4](https://www.ipa4fun.com/ipa/68834/))
```

Comment:
> Update 1.4 states that it adds 4 new locations, these are likely 4 of London, Edinburgh, Copenhagen, Gothenburg, Grimsby. If it's possible to figure out which 4, it may be beneficial to add that info to this list

I don't remember how unlocking cities works in the game, but it may be possible to assert this assumption based on the unlocking order. Either way, I think it's a fair assumption for us to make, so long as we specify that we're making that assumption.

```suggestion
2. "UK & Scandinavia" (Assumed to be Copenhagen, Edinburgh, Gothenburg, Grimsby,  London) [[1.4](https://www.ipa4fun.com/ipa/68834/)]
```

I'm also replacing the `()` with `[]` so that the versioning is different from the annotation.